js/src/as-report.js

```diff
@@ -115,6 +135,11 @@ class ActionsReport extends Component {
 		this.unsetBusy();
 	}
 
+	/**
+	 * Set the busy state flag indicating there is an api call in progress.
+	 *
+	 * @returns {boolean} Flag indicating whether it is safe to call the api.
```

Comment:
This seems odd to return a value "indicating whether it is safe to call the api" from a set/unset function. I would expect two functions `setBusy( true/false )` and `isBusy()` to be more straightforward than (I suspect) checking the result of `setBusy()` and not proceeding if the return value is false.

Obviously this is out of scope of the PR, but worth a mention.

Reply:
@psealock Thanks for the suggestions. The missing types are addressed in https://github.com/woocommerce/action-scheduler-admin/pull/35/commits/711f6e9213aba6bdd8c603187ce7767019a7cedb.

> I would expect two functions setBusy( true/false ) and isBusy() to be more straightforward

I'll address this in a follow up PR.